benchmarking/evaluations/metrics/num_teams.py

```diff
@@ -0,0 +1,7 @@
+from api.dataclasses.team_set import TeamSet
+from benchmarking.evaluations.interfaces import TeamSetMetric
+
+
+class NumberOfTeams(TeamSetMetric):
```

Comment:
Note: this is not really a metric, but really handy to check the number of teams in 1 team set, especially when algorithms do not return the same number of teams